### DISCUSSION TITLE
How to Install Publii in a subdirectory ?

### DISCUSSION THREAD

**Discussion by beartester34** [GetPublii/Publii#1685] [Help] [answered]
Hello everyone. As the title suggests, I want to install Publii in a subdirectory. Like

example.co/publii

is this possible ?

**Comment by michaelfluegge** [comment]
https://getpublii.com/blog/release-046.html :

"_If desired, you can move the previous index page with the post listing to a different directory, such as "blog," by adjusting the settings in the "Pretty URLs" section of your site settings._"

**Reply by bjazmoore** [reply]
> https://getpublii.com/blog/release-046.html :
> 
> "_If desired, you can move the previous index page with the post listing to a different directory, such as "blog," by adjusting the settings in the "Pretty URLs" section of your site settings._"

I do not think that is what beartester34 wants to do.  They have a domain and they want to host publii in a path one level below the domain root and then access the site using the URL example.co/publii.

**Reply by beartester34** [reply]
> > https://getpublii.com/blog/release-046.html :
> > "_If desired, you can move the previous index page with the post listing to a different directory, such as "blog," by adjusting the settings in the "Pretty URLs" section of your site settings._"
> 
> I do not think that is what beartester34 wants to do. They have a domain and they want to host publii in a path one level below the domain root and then access the site using the URL example.co/publii.

Yes, this is exactly what I want to do. I want to host a landing page at my domain, example.com and then host publii, in the folder, example.com/publii

Any pointers are welcome.

**Reply by candidexmedia** [reply]
@beartester34 Will your landing page be made in Publii as well?

**Reply by michaelfluegge** [reply, marked as answer]
Have you ever tried what I suggested? 
I built several 'blogs' with Publii, each in a sub-directory on the same domain.
domain.tld/blog1
domain.tld/blog2
and so on
And I built a landing page (with Publii, in a sub-directory, too) leading to each of the blogs.

An example can be seen here: fluegge.net

**Reply by beartester34** [reply]
> Have you ever tried what I suggested? I built several 'blogs' with Publii, each in a sub-directory on the same domain. domain.tld/blog1 domain.tld/blog2 and so on And I built a landing page (with Publii, in a sub-directory, too) leading to each of the blogs.
> 
> An example can be seen here: fluegge.net

No, I didn't try what you suggested.

Let me try it and if I run into problems, I will reach out to you again.


So, if I understand correctly, I can upload a landing page in the domain , example.com and then , I can upload publii's contents in the directory, example.com/blog and also make the changes in the url settings of Publii.

**Reply by michaelfluegge** [reply]
Yes, that's correct. 
I adapted https://html5up.net/dimension for my landing page and in the navigation I put 
```<nav>
                <ul>
                    <li><a href=" https://fluegge.net/ohnein/">Oh nein!</a></li>
                    <li><a href="https://fluegge.net/wunderland">Wunderland</a></li>
                    <li><a href="https://fluegge.net/fotos">Fotos</a></li>
                    <li><a href="https://fluegge.net/galleries">Galerien</a></li>
                    <li><a href="https://fluegge.net/unterwegs">Unterwegs</a></li>
                    <!-- <li><a href="https://fluegge.net/">Contact</a></li> -->
                    <!--<li><a href="#elements">Elements</a></li>-->
                </ul>
            </nav>
```
into the code. As you can see, the links point to the subfolders, where the Publii sites sit.

**Reply by beartester34** [reply]
> @beartester34 Will your landing page be made in Publii as well?

No, I want to make my landing page using mobirise. Another CMS that outputs HTML files. 

I store all the files on bunny cdn.

**Comment by gpsblues** [comment]
You can distribute the site using FTP by configuring the server settings correctly.

In this example the site is installed in the /en subfolder (the interface is in Italian but I don't think you'll have any problems).

Correctly configure the site address and the subfolder where to transfer Publii (red rectangles).

You don't have to do anything else. The menu and internal link addresses will be corrected automatically.

![image](https://github.com/user-attachments/assets/0d859034-5810-40d3-8c62-3b6abbfcfa89)

**Comment by beartester34** [comment]
I did something else. I wanted to use a good landing page that can have whatever I want in it. Like testimonials, accordions, and everything else that I used to have in wordpress.

I just installed worpdress locally, used this plugin
`Export WP Page to Static HTML/CSS`
exported the homepage of wordpress , using this plugin and then uploaded all the contents in the root folder, i.e., example.com. 

Publii is in example.com/publii

This works perfectly for me and I have various options to tweak the landing page that I made with wordprss.

have a look

![Screenshot 2024-10-12 at 1 37 43 PM](https://github.com/user-attachments/assets/b6e524d9-e03a-43d1-b8c3-6e6a65559341)
